.prettierrc.json

```diff
@@ -1 +1,15 @@
-{}
+{
+  "arrowParens": "avoid",
```

Comment:
The main selling point of Prettier is "Put aside your personal opinion, avoid team arguments, just use the industry standards that we've set". Should we really adjust its defaults?